crates/rattler_upload/src/upload/opt.rs

```diff
@@ -166,6 +169,19 @@ pub struct UploadOpts {
     /// Common options.
     #[clap(flatten)]
     pub common: CommonOpts,
+
+    #[clap(skip)]
+    pub auth_store: Option<Result<AuthenticationStorage, AuthenticationStorageError>>,
```

Comment:
Same comment, this should not be a result. Just `AuthenticationStorage`.

Reply:
Solved!